app/scripts/views/settings/avatar_change.js

```diff
@@ -53,7 +60,7 @@ function ($, Cocktail, FormView, AvatarMixin, SettingsMixin, Template,
 
     remove: function () {
       var self = this;
-      var account = self.getSignedInAccount();
```

Comment:
I have to admit that I have questions about the difference between `getSignedInAccount` and `getAccount` and the need for both, but outside of this PR.